RevenueCatUI/CustomerCenter/Views/NoSubscriptionsView.swift

```diff
@@ -41,25 +44,26 @@ struct NoSubscriptionsView: View {
 
     var body: some View {
         VStack {
-            Text("No Subscriptions found")
-                .font(.title)
-                .padding()
-            Text("We can try checking your Apple account for any previous purchases")
-                .font(.body)
-                .padding()
-
-            Spacer()
-
-            Button("Restore purchases") {
-                showRestoreAlert = true
+            if let localization {
```

Comment:
At this point `localization` should never be null... We could potentially inject a non-nullable parameter in #3968 @vegaro so we don't need to do this here, since at this point, the configuration from the backend is always loaded. Wdyt?